src/sentry/utils/avatar.py

```diff
@@ -88,3 +91,17 @@ def get_letter_avatar(display_name, identifier, size=None, use_svg=True):
             '{initials}</span>').format(color=color, initials=initials,
                                         size_attrs=size_attrs, font_size=font_size,
                                         line_height=line_height)
+
+
+def get_email_avatar(display_name, identifier, size=None):
+    try:
+        validate_email(identifier)
+    except ValidationError:
+        pass
+    else:
+        resp = requests.get(get_gravatar_url(identifier, default=404))
```

Comment:
I'd just catch `Exception`. There's no value in catching specific things, we just don't want anything to fail here ever.